common/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("java-library")
```

Comment:
do we need it here?

considering that `com.hedera.block.common` plugin also has it?

Reply:
I did this based on usages in other places that I saw. In the next part of the epic I will try to see if it is fine when removed along with writing some tests, if it is fine I will include it. Thanks for the note! :)